### PR TITLE
Closes #1011 Add DKT Atlas

### LINKED ISSUE #1011: Add Desikan-Killiany Freesurfer atlas
### Description

Add Desikan-Killiany Freesurver atlas template to xASL.

### Release notes

Add Desikan-Killiany atlas : https://github.com/GGonEsc/petpve12/tree/master/Atlases

### PR DESCRIPTION
**Pull request by MichaelStritt**
### Linked issue

Check out #1011

### How to test

Use the atlas with the **TestDataset** or another subject and run the **Population Module** with the new atlas.

### Comments

Optional: add helpful comments for the reviewers here

### CODE REVIEW

**Comment by MichaelStritt** [comment]
@MDijsselhof: We still need to fill in the labels and maybe we should resample it to our standard space.

**Comment by HenkMutsaerts** [comment]
@MDijsselhof are you certain that they used the Desikan atlas in the previous paper?
I will add AM's converted atlas here @amwink

File: Functions/xASL_init_FileSystem.m [hunk 1]
```diff
@@ -186,6 +186,7 @@
 x.P.Atlas.Tatu_ICA_PCA                      = fullfile(x.D.MapsSPMmodifiedDir, 'VascularTerritories', 'TatuICA_PCA.nii.gz');
 x.P.Atlas.Tatu_ICA_L_ICA_R_PCA              = fullfile(x.D.MapsSPMmodifiedDir, 'VascularTerritories', 'LabelingTerritories.nii.gz');
 x.P.Atlas.Tatu_ACA_MCA_PCA_Prox_Med_Dist    = fullfile(x.D.MapsSPMmodifiedDir, 'VascularTerritories', 'ATTbasedFlowTerritories.nii.gz');
+x.P.Atlas.DKT                  = fullfile(x.D.AtlasDir, 'Desikan_Killiany_MNI_SPM12.nii');
```
Comment (MichaelStritt) [inline Functions/xASL_init_FileSystem.m:189]: This shouldn't be necessary. If the files are in the atlas directory, they should be found automatically.

File: External/Atlases/Desikan_Killiany_License.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+================================================ LICENSE ================================================
```
Comment (MichaelStritt) [inline External/Atlases/Desikan_Killiany_License.txt:1]: We tried to use the exact same name for the license txt files and only to add the postfix _License so far. Check out the other examples. Easy fix 👍

**Review by MichaelStritt — COMMENTED** [review]
Check out my comments from above 👍

File: Functions/xASL_init_FileSystem.m (hunk 1 above)
Comment (MDijsselhof) [reply Functions/xASL_init_FileSystem.m:189]: Okay but how do we tell xASL (using the datapar) that we want to use this atlas?

Comment (MichaelStritt) [reply Functions/xASL_init_FileSystem.m:189]: It uses the filename. So if the atlas is called `Desikan_Killiany_MNI_SPM12` you just have to use `Desikan_Killiany_MNI_SPM12` in the `dataPar.json`.

Comment (MichaelStritt) [reply Functions/xASL_init_FileSystem.m:189]: We only do it differently for the atlases that are located in the SPM modified directory. We probably can't just move them, even though that would be my preferred solution.

Comment (MDijsselhof) [reply Functions/xASL_init_FileSystem.m:189]: Okay!

**Review by HenkMutsaerts — APPROVED** [review]
Nice!

**Pull request merged**